src/patternfly/components/DataList/data-list-toggle-icon.hbs

```diff
@@ -1 +1,6 @@
-<i class="fas fa-angle-right pf-c-data-list__toggle-icon"></i>
+<div class="pf-c-data-list__toggle-icon{{#if data-list-toggle-icon--modifier}} {{data-list-toggle-icon--modifier}}{{/if}}"
```

Comment:
Could this be a `<span>`?

Reply:
yes but i have to add `display: block` so i thought it was better to keep as a `<span>`